### DISCUSSION TITLE
What can i improve on?

### DISCUSSION THREAD

**Discussion by ghost** [community/community#59840] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Show & Tell

### Body

My websites code is at https://github.com/pytutorials/pytutorials.github.io/tree/main

**Comment by leonden** [comment]
Your page looks pretty decent for a tutorial page, I like it. It doesn't have to be crazily designed with some exaggerated elements.
What I would improve though, is the tutorial page because it's not really that user friendly. Just a little subtle styling here and there. Perhaps even similar as the start page.
Hope that helps 😄
